Clinical-Genomics/scout
# Redirect gracefully to variants list when a variant is not found

Fix #2915 

**How to test**:
1. Locally (or on clinical-db stage)
2. Load the demo cancer case.
3. Go to the usual demo variant: http://localhost:5000/cust000/643594/4c7d5c70d955875504db72ef8e1abe77
4. In the URL substitute "643594" with "cancer_case"
5. On master branch the page should crash
6. On this branch, you should be redirected to the cancer case variantS

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
